VelocidiSDK/VelocidiSDK/VelocidiSDK.modulemap

```diff
@@ -1,5 +1,5 @@
 framework module VelocidiSDK {
-    umbrella header "VelocidiSDK.h"
+    umbrella header "VelocidiSDK-umbrella.h"
```

Comment:
Why this change? I'm curious as I was reading on umbrella headers and couldn't find an explanation 🙂

Reply:
No specific reason, I just found it to be common to name this `<Framework's name>-umbrella.h` so I changed it.